managed/services/telemetry/datasource_grafana_sqlitedb_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package telemetry provides telemetry functionality.
+package telemetry
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDatasource(t *testing.T) {
+	t.Parallel()
+	logger := logrus.StandardLogger()
+	logger.SetLevel(logrus.DebugLevel)
+	logEntry := logrus.NewEntry(logger)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*100)
+	t.Cleanup(func() {
+		cancel()
+	})
+
+	config := &Config{
+		ID:      "test",
+		Source:  "GRAFANADB_SELECT",
+		Query:   "count(*) AS total from user",
+		Summary: "Simple query",
+		Data: []ConfigData{
+			{
+				MetricName: "total_users_in_database",
+				Column:     "total",
+			},
+		},
+	}
+
+	t.Run("get metrics from db", func(t *testing.T) {
+		t.Parallel()
+		databaseFile, err := filepath.Abs("../../testdata/telemetry/grafana_sqlite.db")
+		assert.NoError(t, err)
```

Comment:
It has no sense to continue this test if you got an error here.
```suggestion
		require.NoError(t, err)
```

Reply:
Changed.